tests/cucumber/steps/steps.js

```diff
@@ -5419,6 +5426,29 @@ module.exports = function getSteps(options) {
     }
   );
 
+  Then(
+    'the current application should have the following boxes {string}.',
+    async function (boxNames) {
+      let boxes;
+      if (boxNames !== '') {
+        boxes = splitAndProcessAppArgs(boxNames);
+      } else {
+        boxes = [];
+      }
+
+      const resp = await this.v2Client
+        .getApplicationBoxes(this.currentApplicationIndex)
+        .do();
+      const actualBoxes = new Set();
+      for (let i = 0; i < resp.boxes.length; i++) {
```

Comment:
optional nit: could be a map for conciseness

Reply:
I think the suggestion looks better so I implemented it - thanks!